content/getting-started/tutorial/protobuf-client-tutorial.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Writing your first IceRPC + Protobuf client in C#
+---
+
+This tutorial is the second part of a two part series that shows how to create a
+complete application with IceRPC + Protobuf for C#. We start from scratch—you just need to
+have the .NET 8 SDK installed on your computer.
+
+The first part of this tutorial showed how to
+[create the server][protobuf-server-tutorial]. This part shows how to create the client.
+
+{% step title="Create the client" %}
+
+```shell
+dotnet new icerpc-protobuf-client -o MyProtobufClient
+```
+
+This command creates a new IceRPC client application in directory `MyProtobufClient`.
+
+![MyProtobufClient in Visual Studio Code](/images/MyProtobufClient.png)
+
+Let's examine each file:
+
+### proto/greeter.proto - the contract
+
+This file is (and must be) identical or almost identical to the `greeter.proto`
```

Comment:
This sentence sounds like an oxymoron. We should just state if it's identical or not.

```suggestion
This file is almost identical to the `greeter.proto`
```

Reply:
We have the same sentence in the Slice tutorial, should we update it too?

Reply:
My suggesting was: `This file is almost identical to the `greeter.proto``

Reply:
Yeh I agree with @externl , we should update it in both. Just sounds a bit clunky currently.